### PR TITLE
Fix part #1071: High-fi of Topic in progress Portrait mode.

### PR DESCRIPTION
**Pull request by veena14cs**
## Explanation
This PR focuses on hi-fi part of #1071 topic in progress portrait mode.

### Mock
https://xd.adobe.com/view/e8aa4198-3940-47f9-514a-f41cc54457f6-9e9b/screen/a80d1be8-d4b4-4fee-a46d-296b5aa6a110/UP-Topics-in-Progress-/

- [x] Passes Accessibility test

### Screenshot
Screenshot_20200515-152956

### CODE REVIEW

**Comment by veena14cs** [comment]
@mschanteltc I need confirmation on whether we need to show it as a story count or lesson count. In mocks it is `number of lessons` and in our implementation it is `number of stories`.

File: app/src/main/res/layout/ongoing_topic_item.xml [hunk 1]
```diff
@@ -14,68 +14,80 @@
...
+          android:textColor="@color/white_80"
+          android:textSize="14sp"
+          android:textStyle="italic"
+          app:layout_constraintStart_toStartOf="parent"
+          app:layout_constraintTop_toBottomOf="@+id/topic_name_text_view" />
+      </androidx.constraintlayout.widget.ConstraintLayout>
     </androidx.constraintlayout.widget.ConstraintLayout>
   </com.google.android.material.card.MaterialCardView>
 </layout>
+
```
Comment (rt4914) [inline app/src/main/res/layout/ongoing_topic_item.xml]: Remove this extra line.

**Review by rt4914 — APPROVED** [review]
LGTM, just clear your doubt with Chantel and then merge it.

File: app/src/main/res/layout/ongoing_topic_item.xml (hunk 1 above)
Comment (veena14cs) [reply app/src/main/res/layout/ongoing_topic_item.xml]: Done.

**Comment by nikitamarysolomanpvt** [comment]
Please assign me once the requirement is clear.

**Comment by mschanteltc** [comment]
> @mschanteltc I need confirmation on whether we need to show it as a story count or lesson count. In mocks it is `number of lessons` and in our implementation it is `number of stories`.

This will be the number of lessons.

**Comment by veena14cs** [comment]
> > @mschanteltc I need confirmation on whether we need to show it as a story count or lesson count. In mocks it is `number of lessons` and in our implementation it is `number of stories`.
> 
> This will be the number of lessons.

Okay thanks for confirming.

**Comment by veena14cs** [comment]
> > > @mschanteltc I need confirmation on whether we need to show it as a story count or lesson count. In mocks it is `number of lessons` and in our implementation it is `number of stories`.
> > 
> > 
> > This will be the number of lessons.
> 
> Okay thanks for confirming.

Updated to number of lessons.

**Review by nikitamarysolomanpvt — APPROVED** [review]
LGTM

**Pull request merged**